resolver/src/main/java/io/netty/resolver/InetNameResolver.java

```diff
@@ -40,32 +37,15 @@
      */
     protected InetNameResolver(EventExecutor executor) {
         super(executor);
-        loopbackAddress = SocketUtils.loopbackAddress();
     }
 
     /**
+     * @deprecated Use {@link SocketUtils#loopbackAddress()}.
```

Comment:
This method should be an internal detail of `DnsNameResolver` and shouldn't be publicly exposed at this level. I would like to remove it but my assumption is this would be an API breaking change ... it is not clear if I can make API changes to this package because we forgot to put `@UnsableApi` annotations. wdyt?